### PR TITLE
Basic test for AudioListener Attributes

### PR DESCRIPTION
**Pull request by renchenglei**
Automation methods for the position, up, and forward vectors of the AudioListener are added in chromium 52. Add the basic tests for the [three new attributes](https://webaudio.github.io/web-audio-api/#AudioListener).

### CODE REVIEW

File: webaudio/the-audio-api/the-audiolistener-interface/audiolistener-basic.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+<meta charset=utf-8>
+<title>Basic test for AudioListener Attributes</title>
+<script src="/resources/testharness.js></script>
+<script src="/resources/testharnessreport.js></script>
+<script>
+    test(function () {
+        var context = new AudioContext();
+        var listener = context.listener;
+        assert_true("positionX" in listener, "Check if listener.positionX attribute exists");
+        assert_equals(typeof listener.positionX, "object", "Check if listener.positionX attribute is AudioParam type");
```
Comment (jdm) [inline webaudio/the-audio-api/the-audiolistener-interface/audiolistener-basic.html]: Why not use `listener.positionX instanceof AudioParam` instead?

**Comment by jdm** [comment]
Looking further at this test, it seems to be reimplementing by hand many of the features of the existing IDL test harness (eg: https://github.com/renchenglei/web-platform-tests/blob/3a602edf0b7430cddd45a6e8dbc4dcc8eda99afd/webaudio/the-audio-api/the-delaynode-interface/idl-test.html). I think we should probably make use of that instead of writing all of these checks by hand, since the result is more maintainable if the specification changes.

**Comment by jdm** [comment]
More documentation on the IDL test harness is at http://testthewebforward.org/docs/testharness-idlharness.html .

**Comment by renchenglei** [comment]
@jdm Ok, thanks, I will close this pr. I just curious whether it need add idl test to the WebAudio?

**Comment by jdm** [comment]
Sorry, I'm not sure if I understood the question. There does not appear to be a single IDL test for the full WebAudio spec right now; there are separate tests for specific interfaces instead.

**Comment by renchenglei** [comment]
@jdm So sorry. The test cases for Web Audio is so little. Should we add more cases to it?

**Comment by jdm** [comment]
That would be appreciated, yes.

**Pull request closed without merging**